[K9VULN-3850] Remove Rust as supported SCA language (#29353)
DataDog/documentation: content/en/security/code_security/software_composition_analysis/_index.md

Datadog SCA uses a curated proprietary database. The database is sourced from Open Source Vulnerabilities (OSV), National Vulnerability Database (NVD), GitHub advisories, and other language ecosystem advisories, as well as Datadog's own Security Research team's findings. There is a maximum of 2 hours between when a new vulnerability is published and when it appears in Datadog, with emerging vulnerabilities typically appearing in Datadog within minutes.

## Set up Software Composition Analysis
The following languages are supported:
- Python
- JavaScript
- Java
- Csharp
- Go
- Ruby
- PHP

> It is also possible to [upload a third-party SBOM to Datadog][13]

Reviewer: ```suggestion
You can also [upload a third-party SBOM to Datadog][13].
```
